workshops/forms.py

```diff
@@ -71,3 +71,11 @@ class SearchForm(forms.Form):
     in_persons = forms.BooleanField(label='in persons',
                                     required=False,
                                     initial=True)
+
+class DebriefForm(forms.Form):
+    '''Represent general debrief form.'''
+
+    begin_date = forms.CharField(label='Begin date as YYYY-MM-DD',
+                           max_length=10)
+    end_date = forms.CharField(label='End date as YYYY-MD-DD',
```

Comment:
On Sun, Mar 08, 2015 at 12:36:00PM -0700, Piotr Banaszkiewicz wrote:
> We can go pro and use a custom validator
> https://docs.djangoproject.com/en/1.7/ref/validators/#writing-validators
> to make sure a valid date is provided. But I don't see it as a
> necessity.

I've also proposed Bootstrap's datepicker (see #110) for handling date
input.  It's harder to break your input if you're just clicking on
calendar entries.